kafka/common.go

```diff
@@ -215,6 +216,21 @@ func (cfg *CommonConfig) finalize() error {
 		if os.Getenv("KAFKA_TLS_INSECURE") == "true" {
 			cfg.TLS.InsecureSkipVerify = true
 		}
+		if caCertPath := os.Getenv("KAFKA_TLS_CA_CERT_PATH"); caCertPath != "" {
+			caCert, err := os.ReadFile(caCertPath)
```

Comment:
Do we need to support reloading the CA cert file, in case of rotation?

Reply:
true, I think in this case, we may need to close the client and trigger a pod restart, since I don't think there are the mechanisms in place for hot reloading atm.

Reply:
One option I think would be to create a Dialer that dynamically loads the CA cert each time it dials. I haven't thought through that deeply though